source/dotnet/Library/AdaptiveCards/ParseContext.cs

```diff
@@ -5,24 +5,18 @@
 
 namespace AdaptiveCards
 {
-    static class ParseContext
+    public class ParseContext
     {
-        public static void Initialize()
-        {
-            elementIds = new Dictionary<string, List<AdaptiveInternalID>>();
-            idStack = new Stack<Tuple<string, AdaptiveInternalID, bool>>();
-        }
-
         public enum ContextType { Element, Action };
 
-        public static ContextType Type { get; set; }
+        public ContextType Type { get; set; }
 
-        private static IDictionary<string, List<AdaptiveInternalID>> elementIds = new Dictionary<string, List<AdaptiveInternalID>>();
+        private IDictionary<string, List<AdaptiveInternalID>> elementIds = new Dictionary<string, List<AdaptiveInternalID>>();
```

Comment:
This file is also missing hte copyright header. Could you add it in? thanks!

Reply:
sure

---
In reply to: [341394563](https://github.com/microsoft/AdaptiveCards/pull/3556#discussion_r341394563) [](ancestors = 341394563)